### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Local IIS Add Server_Keyboard:focus is not visible on "Remember this server" check box

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#78]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)Chrome(68.0.3440.106)
Keyboard.
MAS ID Violated: MAS 2.4.7

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Local IIs link on the top navigation bar and press enter
4.Tab till Remember this server check box
4.Check whether the focus  is visible on check box or not

Actual Result:
focus is not visible on "Remember this server" check box

Expected Result:
Keyboard focus should visible on the check box

User Impact:
Keyboard only users find difficulty in using the application if focus is not visible on the particular control

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
![bug_78_focus is not visible on check box](https://user-images.githubusercontent.com/42638683/44581402-2c5fd100-a7bb-11e8-9f55-efd8285b091d.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.7; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
